ocaml/idl/datamodel_errors.ml

```diff
@@ -411,6 +411,8 @@ let _ =
     ~doc:"An error occured while restoring the memory image of the specified virtual machine" ();
   error Api_errors.vm_pv_drivers_in_use [ "vm" ]
     ~doc:"VM PV drivers still in use" ();
+  error Api_errors.invalid_VCPU_max ["field"; "value"]
```

Comment:
The field seem redundant because always the same field is used but it does no harm.

Reply:
I think replacing the field with `cores-per-socket` would allow the UI to inform the user what's wrong and how to fix it:
```
error Api_errors.vcpu_max_not_cores_per_socket_multiple ["vcpu_max"; "cores_per_socket"]
```
which the can be used as
```
Api_errors.(Server_error vcpu_max_not_cores_per_socket_multiple [vcpus; cpds])
```

Reply:
@psafont's suggestion is better.

Reply:
Thank you so much. I will update it now. :)